client/src/pages/AllProjrcts.jsx

```diff
@@ -0,0 +1,24 @@
+import { Link } from "react-router-dom";
+import ShowAllProjects from "../components/showAllProjects/ShowAllProjects";
+import styled from "styled-components";
+import AddProject from "./AddProject";
+
+const AllProjects = ({allProjects}) => {
+    return (
+        <Container>
+            <Link to='/addproject'><button>add new project</button></Link>
```

Comment:
You don't need a `button` element. You can work with `Link` as you already do and if you need to style the `Link` you can change in line 20 the `button` to `a` or make a new styled component:

```
const StyledLink = styled(Link)`
  margin-top: 2em;
`
```